fastplotlib/graphics/_base.py

```diff
@@ -264,22 +285,21 @@ class PreviouslyModifiedData:
     indices: Any
 
 
+COLLECTION_GRAPHICS: dict[str, Graphic] = dict()
+
+
 class GraphicCollection(Graphic):
     """Graphic Collection base class"""
 
     def __init__(self, name: str = None):
         super(GraphicCollection, self).__init__(name)
-        self._graphics: List[Graphic] = list()
-
-    @property
-    def world_object(self) -> Group:
-        """Returns the underling pygfx WorldObject."""
-        return self._world_object
+        self._graphics: List[str] = list()
 
     @property
     def graphics(self) -> Tuple[Graphic]:
-        """returns the Graphics within this collection"""
-        return tuple(self._graphics)
+        """The Graphics within this collection. Always returns a proxy to the Graphics."""
+        proxies = [weakref.proxy(COLLECTION_GRAPHICS[loc]) for loc in self._graphics]
+        return tuple(proxies)
```

Comment:
`self._graphics` houses all of the locs for each graphic in a collection, so then when a user accesses `self.graphics` it will return a weak proxy to each of the graphics that are being indexed for?

Reply:
yup!

Reply:
would probably be easier to read by unfurling the list comprehension